packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -27,9 +28,10 @@ import ListDiffer, { DiffResult } from '@egjs/list-differ';
     ':host {display: block}', ':host > div {width: 100%; height: 100%}'
   ]
 })
-export class NgxFlickingComponent implements OnInit, AfterViewInit, OnDestroy, OnChanges {
+export class NgxFlickingComponent implements OnInit, AfterViewInit, OnDestroy, OnChanges, AfterViewChecked, DoCheck {
   @Input() options: Partial<FlickingOptions> = {};
   @Input() plugins: Plugin[] = [];
+  @Input() panels: [];
```

Comment:
I think this code regards the type of `panels` as an empty array.
If you intended to set `panels` can be anything, I think `any[]` is a more proper way to do that.

Reply:
@WoodNeck Yes, panels can be anything as you said. I'll fit it. Thank you!